From 56b0548c5ef64bb466e60665db3c4940c2b6daa9 Mon Sep 17 00:00:00 2001
Message-Id: <56b0548c5ef64bb466e60665db3c4940c2b6daa9.1414996573.git.chang-joon.lee@intel.com>
In-Reply-To: <de6d11b8587be186d089e692fc410008d4d87034.1414996573.git.chang-joon.lee@intel.com>
References: <de6d11b8587be186d089e692fc410008d4d87034.1414996573.git.chang-joon.lee@intel.com>
From: Rafael Barbalho <rafael.barbalho@intel.com>
Date: Tue, 28 Oct 2014 17:07:38 +0000
Subject: [PATCH 02/10] FOR_UPSTREAM [VPG]: drm/i915: Remove unused variable
 in userptr code

Fix the fumble in:

Author: Tvrtko Ursulin <tvrtko.ursulin@intel.com>
Date:   Fri Sep 26 15:05:22 2014 +0100

    FOR_UPSTREAM [VPG]: drm/i915: Do not leak pages when freeing userptr objects

For: GMINL-1778
Change-Id: I3380d4745a20afc90872dde984225ea24bdbc090
Signed-off-by: Rafael Barbalho <rafael.barbalho@intel.com>
---
 drivers/gpu/drm/i915/i915_gem_userptr.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/i915_gem_userptr.c b/drivers/gpu/drm/i915/i915_gem_userptr.c
index bcea097..1cd9fa6 100644
--- a/drivers/gpu/drm/i915/i915_gem_userptr.c
+++ b/drivers/gpu/drm/i915/i915_gem_userptr.c
@@ -681,7 +681,6 @@ i915_gem_userptr_get_pages(struct drm_i915_gem_object *obj)
 static void
 i915_gem_userptr_put_pages(struct drm_i915_gem_object *obj)
 {
-	struct scatterlist *sg;
 	struct sg_page_iter sg_iter;
 
 	BUG_ON(obj->userptr.work != NULL);
-- 
1.7.9.5

